wodles/aws/subscribers/s3_log_handler.py

```diff
@@ -305,3 +305,101 @@ def process_file(self, message_body: dict) -> None:
         for event in events_in_file:
             self.send_msg(event, dump_json=False)
         aws_tools.debug(f'{len(events_in_file)} events sent to Analysisd', 2)
+
+
+class AWSSecurityHubSubscriberBucket(AWSSubscriberBucket):
+    """Class for processing events from AWS S3 buckets containing Security Hub related logs."""
+
+    @staticmethod
+    def _add_event_type_fields(details: dict, event: dict):
+        """
+        Add the corresponding fields into the event if the details contain them.
+
+        Parameters
+        ----------
+        details : dict
+            Source dictionary containing the events from the log file.
+        event : dict
+            Destination dictionary to be added to the event sent to Wazuh.
+        """
+        key_actions = {
+            'findings': lambda source, destination: destination.setdefault('finding', source['findings'][0]),
```

Comment:
The default value should be `finding`?

```suggestion
            'findings': lambda source, destination: destination.setdefault('findings', source['findings'][0]),
```

Reply:
Although the log files contain the `findings` key, it always returns a single finding inside its list. That's why the name of the key is `finding`.